admob/src/android/banner_view_internal_android.cc

```diff
@@ -120,7 +117,8 @@ struct BannerViewInternalInitializeData {
   BannerViewInternalInitializeData()
       : activity_global(nullptr),
         ad_view(nullptr),
-        banner_view_helper(nullptr) {}
```

Comment:
Just putting this out there, since we don't need to support STLPort any more, you could de-void the internal initialize function and use std::function I think.

Reply:
Sorry, I'm not quite sure I understand. Could we talk about this offline and have it part of the Banner View rewrite (which has it's own story.)

Reply:
SG, let's do this offline. (basically, we can stop using void* context pointers in callbacks, and just use std::function instead.)